### PR TITLE
[CI/CD] fix CD bug, update docs with new installation procedures

### PR DESCRIPTION
**Pull request by zhisbug**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -4,9 +4,61 @@ Install Alpa
...
+- CUDA (CuDNN): 11.1 (8.0.5), 11.2 (8.1.0), 11.3 (8.2.0)
+- Python: 3.7, 3.8, 3.9
+
+If you need to use other CUDA, CuDNN, or Python versions, please follow the next section to :ref:`install from source<install-from-source>`.
+
+1. To install from wheels, first install Alpa:
+
+  .. code:: bash
+
+    pip3 install --upgrade pip
```
Comment (merrymercy) [inline docs/install.rst:25]: Why is this line required?

File: docs/install.rst [hunk 2]
```diff
@@ -80,3 +132,47 @@ You can check the installation by running the following test script.
...
+  >>> import alpa
+    ......
+    RuntimeError: jaxlib version 0.3.7 is newer than and incompatible with jax version 0.3.5. Please update your jax and/or jaxlib packages
+
+Make sure your jax version is 0.3.5, Flax version is 0.4.1 by reinstalling them following:
+
+.. code:: bash
+
+  pip3 install jax==0.3.5
+  pip3 install flax==0.4.1
```
Comment (merrymercy) [inline docs/install.rst:176]: Does this mean we have to update this webpage every time we release the package?

File: docs/install.rst (hunk 1 above)
Comment (zhisbug) [reply docs/install.rst:25]: mimicking https://github.com/google/jax#pip-installation-cpu

File: docs/install.rst (hunk 2 above)
Comment (zhisbug) [reply docs/install.rst:176]: no, why you think so?
